addons/escoria-core/game/core-scripts/resources/esc_animationresource.gd

```diff
@@ -0,0 +1,58 @@
+# Resource containing all defined animations and angles for 
+# characters movement.
+ 
+extends Resource
+class_name ESCAnimationResource
+
+
+# Array containing the different angles available for animations.
+# Each angle is defined by an array [start_angle, angle_size].
+# start_angle must be between 0 and 360.
+# Angle 0 and 360 are the same and correspond to UP/NORTH
+# 90 is RIGHT/EAST, 180 is DOWN/SOUTH, etc
+export(Array, Resource) var dir_angles = []
```

Comment:
If you use `_get_property_list`, you should remove the export keywords here. So this line should be just:

```
var dir_angles: Array = []
```